src/index.js

```diff
@@ -100,7 +100,7 @@ export function run (command, options = {}, logger = loggerAlias) {
 
   // Pick relevant option keys and set default values
   options = {
-    env: options.env || {},
+    env: options.env || process.env,
```

Comment:
Defaulting to `process.env` may lead to inconsistent results, since different developers may have different environment variables set.

In practice, the shell currently defaults to `sh` which supports a relatively small number of environment variables. One that could cause problems is `IFS`, the input file separator variable. However, this is rarely changed from the default value.

If other execution shells are supported in the future like `bash` this could be come a bigger issue, as variables like `SHELLOPTS` and `BASHOPTS` can have a bigger impact on the shell behavior.

`systemd` takes a safer approach of providing a very small default environment and then requiring developers to explicitly declare environment variables that they want to set or pass through. That approach creates a more consistent execution environment.

You may recall experience of a script that worked fine when run manually, but failed when run via `cron`, which had different environment variables set.  `systemd` avoid that problem with a more tightly controlled environment so `systemctl start some-service` will work the same from the CLI has it does when run from a `systemd` timer. 

Here there could be potential to different results on a developer workstation with a customized environment compared to a CI server with a barren environment.